src/uct/ib/base/ib_iface.c

```diff
@@ -82,13 +82,21 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    ucs_offsetof(uct_ib_iface_config_t, tx.cq_moderation), UCS_CONFIG_TYPE_UINT},
 
 #if HAVE_DECL_IBV_EXP_CQ_MODERATION
-  {"TX_CQ_MODERATION_COUNT", "-1",
+  {"TX_CQ_EVENT_MODERATION_COUNT", "65535", /* MAX UINT16 */
```

Comment:
we need a shorter name.. 
how about TX_EVENT_MOD_COUNT, TX_EVENT_MOD_PERIOD?
also, why max_uint16?

Reply:
renamed.
max_int16 is used as default maximum value for moderation value (it operates by uint16 values), but it looks like not very gracefully, updated to process 0 as default value to disable moderation